src/Template/Bake/Template/index.ctp

```diff
@@ -39,20 +39,21 @@ $fields = collection($fields)
     endforeach;
 %>
     </ul>
-</div>
-<div class="<%= $pluralVar %> index large-10 medium-9 columns">
+</nav>
+<div class="<%= $pluralVar %> index large-9 medium-8 columns" id="content">
```

Comment:
There is no semantic reason usually for classes (stuff like http://en.wikipedia.org/wiki/Microformat aside).

As @mtkocak already pointed out: Foundation is not bundled and classes (and mixins) may miss so @ADmad there is no easy way to add new elements as required by the app.

If I get to it I will purpose a fix that simply uses css selectors for the same set of properties instead of classes.